google-ads-examples/src/main/java/com/google/ads/googleads/examples/accountmanagement/GetAccountHierarchy.java

```diff
@@ -42,11 +42,12 @@
 import java.util.Set;
 
 /**
- * Gets the account hierarchy of the specified manager account. If you don't specify manager ID, the
- * example will instead print the hierarchies of all accessible customer accounts for your
- * authenticated Google account. Note that if the list of accessible customers for your
- * authenticated Google account includes accounts within the same hierarchy, this example will
- * retrieve and print the overlapping portions of the hierarchy for each accessible customer.
+ * Gets the account hierarchy of the specified manager account and login customer ID. If you don't
+ * specify manager ID or login customer ID, the example will instead print the hierarchies of all
```

Comment:
Small question: should it be "manager ID and login customer ID"?
We want the user to provide both or none, right?